### DISCUSSION TITLE
How to organize from config.json or other ? it's any sample ?

### DISCUSSION THREAD

**Discussion by snokin** [filebrowser/filebrowser#1249] [Q&A] [answered]
I didn't see any setting configuration like "disable-preview-resize" or "disable-thumbnails" in github page or the website, But i found these via googling. Is it has more hidden settings like that ? However i found disable thumbnails settings and now it's work on my little chip server smooth. But the cache-dir seens dosn't work ? Is it im doing wrong ? Here is my config.json:

> {
	"address":"0.0.0.0",
	"database":"/etc/filebrowser/filebrowser.db",
	"disable-preview-resize":"true",
	"disable-thumbnails":"true",
	"log":"/var/log/filebrowser.log",
	"port":"83",
	"root":"/home/filebrowser",
	"username":"admin",
	"cache-dir":"/home/filebrowser/.cache"
}

**Comment by o1egl** [comment, marked as answer]
`--cache-dir` is a cli only param. At the moment there is no equivalent config param.
